tests/suites/hup/suite.js

```diff
@@ -88,15 +88,6 @@ const doHUP = async (that, test, mode, hostapp, target) => {
 	// send hostapp to DUT
 	await that.worker.sendFile(hostapp, `/mnt/data/resin-data/`, target);
 	const hostappPath = `/mnt/data/resin-data/${hupOsName}`;
-	console.log(hostappPath);
-
-	const balenaHostTmpPath = "/mnt/sysroot/inactive/balena/tmp";
-	const hupLoadTmp = "/mnt/data/resin-data/tmp";
-
-	await that.worker.executeCommandInHostOS(
-		`grep -q "LOADTMP" "$(command -v hostapp-update)" || { mkdir -p "${hupLoadTmp}" "${balenaHostTmpPath}" ; mount --bind "${hupLoadTmp}" "${balenaHostTmpPath}" ; }`,
```

Comment:
The grep here is to check if the hostapp-update script has already been updated to include the mount command. Existing devices in the field will not have the updated script, and will not have space in rootfs, so we need to mount in most cases right now.

Reply:
I see @klutchell I didn't realise, I'll update it

Reply:
In the PR https://github.com/balena-os/meta-balena/pull/2510 I actually check to make sure it's not already a mountpoint, but we should attempt to use the script where possible, hence the grep for `LOADTMP`.